### ISSUE TITLE
macOS 10.13: greadlink: command not found

### ISSUE THREAD

**Issue by felixrabe** [area9innovation/flow9#216]
TLDR: Please add a mention of the (current) requirement to run `brew install coreutils` on macOS to the main README's [installation instructions](https://github.com/area9innovation/flow9/blob/master/README.md#installation).

(**EDIT**: Basically rewrote this description to be more concise and actionable.)

I get this error on macOS:

```
$ git clone https://github.com/area9innovation/flow9.git  # f060aca8d
...
$ cd flow9
$ ./bin/flowcpp ./demos/demos.flow
./bin/flowcpp: line 11: greadlink: command not found
```

Because the [installation instructions](https://github.com/area9innovation/flow9/blob/f060aca8d2b35b80e6572e5d3982b3a6202d25b0/README.md#installation) forgot to mention the dependency on `coreutils`.

This _is_ described in [doc/mac.markdown#install-greadlink](https://github.com/area9innovation/flow9/blob/master/doc/mac.markdown#install-greadlink) though (found via [doc/readme.markdown](https://github.com/area9innovation/flow9/blob/master/doc/readme.markdown)):

```markdown
## Install `greadlink`

This command will install the GNU version of `readlink`, and a bunch
of other GNU utilities.

    brew install coreutils

The GNU versions names' have a `g` prepended, e.g. `greadlink`, so
having them around won't conflict with your Mac's built-in versions.

`greadlink` is needed because the scripts for running Flow assume the
GNU semantics, so they won't work with the standard Mac `readlink`.
(Note that fixing that wouldn't actually be very hard.)
```

**Comment by dmitrys99** [comment]
https://github.com/area9innovation/flow9/blob/da3887649690a1a535ebd9b8289673d49591888a/bin/flowcpp#L11
Please suggest something instead of this, we'll be happy to avoid `greadlink` as well on Mac.

**Comment by felixrabe** [comment]
I'll look into the `greadlink` replacement, because I've done some related Bash scripting stuff in the past and might come up with something. If I do, I'll open a PR.
